src/routes/kickPlayer.ts

```diff
@@ -0,0 +1,92 @@
+import { getClient } from '../utils/databaseConnection'
+import { rateLimiter } from '../utils/rateLimiter'
+import { celebrate, Joi, Segments } from 'celebrate'
+import { verifyFCMToken } from '../utils/firebase'
+
+import express, { type Router } from 'express'
+const router: Router = express.Router()
+
+router.get(
+  '/kickPlayer',
+  rateLimiter,
+  celebrate({
+    [Segments.QUERY]: Joi.object().keys({
+      creatorToken: Joi.string()
+        .required()
+        .min(1)
+        .max(250)
+        .label('creatorToken'),
+      playerToken: Joi.string().required().min(1).max(250).label('playerToken'),
+    }),
+  }),
+  async (req, res) => {
+    if (
+      !(
+        (await verifyFCMToken(req.query.creatorToken)) &&
+        (await verifyFCMToken(req.query.playerToken))
+      ) ||
+      req.query.creatorToken === req.query.playerToken
+    ) {
+      return res.sendStatus(400)
+    }
+
+    const client = getClient()
+    client
+      .connect()
+      .then(async () => {
+        const getGameQuery = 'SELECT game_id FROM Games WHERE game_master=$1'
+        const verifyPlayerInGameQuery =
+          'SELECT token FROM Players WHERE token=$1 AND game_id=$2'
+        const removePlayerFromGameQuery = 'DELETE FROM Players WHERE token=$1'
+        const getNewSmallBlindCurrentPlayer =
+          'SELECT token FROM Players WHERE turn IN (SELECT MIN(turn) FROM Players WHERE token!=$1 ) AND game_id=$2 LIMIT 1'
```

Comment:
I think we'll have to do this a bit differently. Just remove these two blind queries for now and I'll do something with it when making `startGame`. I think we want something like a `fixGameState()` function, that we can call after kicking a player, or after a player leaves on their own. Then the function will also check if the game has already started or not and notify players of the changes.

Let's keep the scope of this PR simple. Just remove the player and assume we're in the lobby stage, as this is what we promised in our second spritnt.